etc/services/osf-localhost.json

```diff
@@ -3,7 +3,7 @@
   "id" : 203948234207230,
   "name" : "",
   "description" : "",
-  "serviceId" : "^https?://(localhost|127.0.0.1|dev.osf.io):500[0-9]/(?!oauth/callback).*",
+  "serviceId" : "^https?://(localhost|127.0.0.1|dev.osf.io):500[0-9]/(?!(oauth/callback|erpc|prereg|(login/?\\?next=https?(%3A|:)(%2F|/)(%2F|/)(localhost|127.0.0.1|dev.osf.io)(%3A|:)500[0-9](%2F|/)preprints(%2F|/)))).*",
```

Comment:
We should be able to avoid all the special cases for other services if we order the evaluation of services preprints -> osf -> ...